docs/source/release/v6.0.0/direct_geometry.rst

```diff
@@ -21,7 +21,27 @@ CrystalField
 New
 ###
 - Extended the ``Background`` class to accept a list of functions through using the ``functions`` keyword. This
-  allows more than two functions to be used for the ``Background`` of a CrystalField fit.
+  allows more than two functions to be used for the ``Background`` of a CrystalField fit as seen in the example below.
+
+.. code-block:: python
+
+    cf.background = Background(functions=[Function('Gaussian',Sigma=0.05313/2, Height= 1279300,PeakCentre=-0.0021),
+                                          Function('Gaussian',Sigma=0.1111/2, Height= 1465,PeakCentre=0.09147),
+                                          Function('LinearBackground', A0=34, A1=0.01)])
+
+    # Ties can then be applied by indexing to the relevant function
+    cf.background.functions[1].ties(Height=1465, Sigma=0.1111/2, PeakCentre=0.09147)
+
+- Implemented a method for fixing function parameters in the ``Background`` to their current values. An example using
+  the same background as above is:
+
+.. code-block:: python
+
+    # Fixes the PeakCentre and Height of the first Gaussian to their current values.
+    cf.background.functions[0].fix('PeakCentre', 'Height')
+
+    # Fixes all the parameters of the LinearBackground to their current values.
+    cf.background.functions[2].fix('all')
```

Comment:
Could we add an adapted version of this to `docs/source/techniques/CrystalField.rst`, something like

```
Tying Background Parameters
=======================

    cf.background = Background(functions=[Function('Gaussian',Sigma=0.05313/2, Height= 1279300,PeakCentre=-0.0021),
                                          Function('Gaussian',Sigma=0.1111/2, Height= 1465,PeakCentre=0.09147),
                                          Function('LinearBackground', A0=34, A1=0.01)])

    # Fixes the PeakCentre and Height of the first Gaussian to their current values.
    cf.background.functions[0].fix('PeakCentre', 'Height')

    # Fixes all the parameters of the LinearBackground to their current values.
    cf.background.functions[2].fix('all')
```

So users have an example of how to do this outside of the release notes?